src/components/application/List.jsx

```diff
@@ -0,0 +1,174 @@
+                students with disabilities, from families with an annual income
+                below ₹2,50,000.
+              </Text>
+              <Text mt={4}>
+                Day scholars receive ₹3500 (₹3850 for disabled students), and
+                hostellers receive ₹7000 (₹7700 for disabled students). The
+                scholarship is available to students domiciled in Madhya Pradesh
+                and is auto-renewed. Students can avail this scholarship even if
+                they are benefiting from another government scholarship, and it
+                is available for one year if the student fails.
+              </Text>
+              <Heading size="md" color="#484848" fontWeight={500} mt={6}>
+                Objectives of the Pre-matric <br />
+                Scholarship-ST:
+              </Heading>
+              <UnorderedList mt={4}>
+                <ListItem>
+                  Provide financial assistance to ST students in Classes 9 and
+                  10 to encourage continued education.
+                </ListItem>
+                <ListItem>
+                  Support low-income families by reducing the financial burden
+                  of schooling.
+                </ListItem>
+                <ListItem>
+                  Promote equal educational opportunities for students with
+                  disabilities through higher financial aid.
+                </ListItem>
+                <ListItem>
+                  Reduce dropout rates among ST students by offering incentives
+                  to complete secondary education.
+                </ListItem>
+                <ListItem>
+                  Ensure educational support for ST students in both urban and
+                  rural areas, particularly in Madhya Pradesh.
+                </ListItem>
+              </UnorderedList>
+              <Heading size="md" color="#484848" fontWeight={500} mt={6}>
+                Key Points:
+              </Heading>
+              <UnorderedList mt={4}>
+                <ListItem>Available to ST students in Madhya Pradesh.</ListItem>
+                <ListItem>Supports day scholars and hostellers.</ListItem>
+                <ListItem>
+                  Annual family income must be below ₹2,50,000.
+                </ListItem>
+                <ListItem>
+                  Application deadline: 31st July, valid till 31st October.
+                </ListItem>
+              </UnorderedList>
+
+              <Heading size="md" color="#484848" fontWeight={500} mt={6}>
+                Mandatory Documents:
+              </Heading>
+              <UnorderedList mt={4}>
+                <ListItem>Marksheet</ListItem>
+                <ListItem>Income Certificate</ListItem>
+                <ListItem>Caste Certificate</ListItem>
+              </UnorderedList>
+            </Box>
+
+            <Box m={4}>
+              <Button className="custom-btn" type="submit" mt={4} width="100%" onClick={openModal}>
```

Comment:
Please use the button from the common component.

Reply:
added from common components